hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.js

```diff
@@ -0,0 +1,70 @@
+import React, { PureComponent } from 'react'
+import styles from './Navbar.module.scss';
+import { Link } from 'react-router-dom';
+import { ReactComponent as Logo } from './../../../logo.svg';
```

Comment:
Don't need any of these `../` once you merge in develop, just `logo.svg`. Same goes for everything below, just `assets/<files>` etc